cookies/views.py

```diff
@@ -1121,10 +1121,13 @@ def create_collection(request):
     """
     context = RequestContext(request, {})
 
+    parent_id = request.GET.get('parent_collection', None)
```

Comment:
Next big thing: we need to make sure that the user has permission to create a sub-collection. You can check this with ``check_authorization`` in the ``authorizations`` module. The authorization we want to check here is "change". Let me know if you run into trouble, it's still a bit wonky sometimes.